### PR TITLE
Vetrina layouts: landing screen

### PR DESCRIPTION
**Pull request by vapaj**
Add #317

Also changed the functionality so that `vetrina` will get the products wished to show through `Props`, not by fetching the things itself.

Also `label` is an optional field of `InputField` now.

### CODE REVIEW

File: apps/vetrina/src/Vetrina/Main.purs [hunk 1]
```diff
@@ -42,7 +38,7 @@ import Vetrina.Purchase.Completed as PurchaseCompleted
 
 foreign import sentryDsn_ :: Effect String
 
-type Props = { onClose :: Effect Unit }
+type Props = { onClose :: Effect Unit, products :: Array Product }
```
Comment (ae-mo) [inline apps/vetrina/src/Vetrina/Main.purs:41]: Why this formatting?

Comment (vapaj) [reply apps/vetrina/src/Vetrina/Main.purs:41]: Well, first of all it makes sense that whoever includes vetrina, also somehow tells it which products to show. Also, we don't want to hard code the descriptions of the products to vetrina, so it makes sense to just take the full products in props.

Comment (ae-mo) [reply apps/vetrina/src/Vetrina/Main.purs:41]: I just meant why the object's items are on the same line 😅

Comment (vapaj) [reply apps/vetrina/src/Vetrina/Main.purs:41]: Oh 🤠 It's such a short definition it fits nicely in one line

Comment (ae-mo) [reply apps/vetrina/src/Vetrina/Main.purs:41]: But apparently it's common in the code, so I won't be bothering anymore 😃

Comment (vapaj) [reply apps/vetrina/src/Vetrina/Main.purs:41]: Yeah, I don't know... I mean
```
type Props =
  { onClose :: Effect Unit }
```
would look a bit funny (although I suspect we have this style in the code as well :D)

But you're right, if there's > 1 attributes, they should be formatted in separate lines:
```
type Props =
  { onClose :: Effect Unit
  , products :: Array Product
  }
```

**Pull request merged**